domains/ela.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "DXRdev",
+        "discordID": "335635478684499970",
+        "discord": "elastrixx"
+    },
+    "record": {
+        "TXT": ["dh=146345133ea568500a36844c85d85ec0aead3bae"]
```

Comment:
Please use a different record, e.g. CNAME for GitHub Pages. A single TXT record that is not going to be accepted.